### ISSUE TITLE
VRaptor 4.1.1, My StringConverter does not work

### ISSUE THREAD

**Issue by rafaelbtz** [caelum/vraptor4#1072]
The String converter is never called, i tried this:
```
@Alternative
@Priority(Interceptor.Priority.APPLICATION)
@Convert(String.class)
public class StringTrimConverter implements Converter<String> {
	@Override
	public String convert(String value, Class<? extends String> type) {
		System.out.println("Converter");
		return value == null || value.length() == 0 ? null : value.trim();
	} 
}
```

and this:
```
@Convert(String.class)
@Specializes
public class StringTrimConverter extends StringConverter {
	@Override
	public String convert(String value, Class<? extends String> type) {
		return value == null || value.length() == 0 ? null : value.trim();
	}
}
```

**Comment by dhpagani** [comment]
Hi! Is there some schedule when it will be fixed?  We are upgrading our application and we need this fixed to avoid refactoring legacy code.

I also tried find where is the bug to correct and send but i couldn't :(

Thank you!

**Comment by Turini** [comment]
@dhpagani we just had some positive updates on vraptor's mail list:
https://groups.google.com/forum/#!topic/caelum-vraptor/98KzmkmDij4
it looks like Julio will open a PR soon

**Comment by jclazzarim** [comment]
Hi guys, I just make a PR to (try!?) help fix this issue. I have made some tests with String arrays and worked fine too.

**Comment by Turini** [comment]
@rafaelbtz @dhpagani could you guys test using this Julio's code on your projects? 

This is the PR: https://github.com/caelum/vraptor4/pull/1078/files

**Comment by dhpagani** [comment]
Hi, I test the code and it had worked like a charm!

**Comment by Turini** [comment]
this is great! I'll merge and release it soon. thank you guys

**Comment by Turini** [comment]
Great! The PR is already merged, I'll release the new version including this code asap.

**Comment by Turini** [comment]
I just released vraptor 4.2 rc5, could you guys try it out?
http://repo1.maven.org/maven2/br/com/caelum/vraptor/4.2.0-RC5/
